dash/include/dash/pattern/BlockPattern1D.h

```diff
@@ -799,7 +818,6 @@ class BlockPattern<1, Arrangement, IndexType>
     /// Global block index
     index_type g_block_index) const
   {
-#if 1
```

Comment:
The deactivated code sections are previous implementations before I rewrote them as constexpr. Left them as comments to help debugging in case my constexpr variant was broken. Will remove them.